### ISSUE TITLE
Script Server: Link pause script to pause run

### ISSUE THREAD

**Issue by ThomasLohnert** [ISISComputingGroup/IBEX#5288]
As a LSS scientist, I would like pausing the script server execution to automatically pause the run. This would avoid confusion stemming from having various "pause" buttons with different meanings. They said they could not think of a case where they would want to pause script execution but keep collecting data.  Perhaps this behaviour could be enabled/disabled as a preference for the script server?

#### Acceptance Criteria
- (Gather feedback from other scientists?)
- Provide option to automatically pause data collection when pausing script execution
- Provide option to automatically resume data collection when resuming script execution

**Comment by kjwoodsISIS** [comment]
Yes, we should definitely check with other groups.  Some may prefer to have data collection keep going.
